src/processes/topic-auto-msg-process.js

```diff
@@ -0,0 +1,75 @@
+const {Op} = require('sequelize');
+const {UserTopic} = require('../databases/models');
+const db = require('../databases/models');
+
+const StreamChat = require('stream-chat').StreamChat;
+require('dotenv').config();
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

It's not a good practice to require and configure dotenv in every file. It should be done once in the main application file (like app.js or server.js). This way, you ensure that environment variables are loaded before any other part of your application starts to execute.
```diff
- const {Op} = require('sequelize');
- const {UserTopic} = require('../databases/models');
- const db = require('../databases/models');
- 
- const StreamChat = require('stream-chat').StreamChat;
- require('dotenv').config();
+ const {Op} = require('sequelize');
+ const {UserTopic} = require('../databases/models');
+ const db = require('../databases/models');
+ 
+ const StreamChat = require('stream-chat').StreamChat;
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->